docs/parsers/JsonParser.md

```diff
@@ -0,0 +1,30 @@
+---
+title: JSON Parser
+author: williamabradley
+ms.date: 01/19/2018
+description: The JSON Parser allows you to parse a JSON content String into JSON Schema.
+keywords: uwp community toolkit, uwp toolkit, microsoft community toolkit, microsoft toolkit, json, json parsing, parser
+---
+
+# JSON Parser
+
+The [JsonParser](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.parsers.json.jsonparser) class allows you to parse a JSON content String into a JSON Schema.
+
+| Class | Purpose |
+| --- | --- |
+| **Microsoft.Toolkit.Parsers.Json.JsonParser** | Parser for Parsing JSON Strings into JSON Schema. |
+| **Microsoft.Toolkit.Parsers.Core.SchemaBase** | Schema for Parsing Content into a desired format. |
+
+## Sample Code
+
+[JSON Parser Sample Page Source](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/JsonParser/JsonParserPage.xaml.cs).
+
+You can see this in action in [UWP Community Toolkit Sample App](https://www.microsoft.com/store/apps/9NBLGGH4TLCQ).
+
+## Requirements
```

Comment:
Requirements needs to follow template structure: https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md

Reply:
Done.